orangecontrib/spectroscopy/preprocess/__init__.py

```diff
@@ -312,6 +312,12 @@ def transformed(self, data):
                                   limits=[[self.lower, self.upper]])(data)
             data.X /= norm_data.X
             replace_infs(data.X)
+        elif self.method == Normalize.MinMax:
+            min_data = np.min(data.X, axis=1)
+            max_data = np.max(data.X, axis=1)
+            norm_data = (data.X.T - min_data) / (max_data - min_data)
```

Comment:
Why all the transposing?

Reply:
Otherwise, the sizes are not compatible.

Reply:
Oh I see. You can also pass `keepdims=True` to min/max for the same effect.